csi/csisanity_test.go

```diff
@@ -127,8 +127,13 @@ func TestCSISanity(t *testing.T) {
 	}
 
 	// Start CSI Sanity test
+	targetPath := "/tmp/mnt/csi"
 	sanity.Test(t, &sanity.Config{
 		Address:    server.Address(),
-		TargetPath: "/mnt",
+		TargetPath: targetPath,
+		CreateTargetDir: func(p string) (string, error) {
+			os.MkdirAll(p+"/target", os.FileMode(0755))
```

Comment:
The CSI mock driver no longer creates this for us, so we need to do this manually with this flag. 

See release notes:
https://github.com/kubernetes-csi/csi-test/releases/tag/v2.0.0